lib/compare_with_wikidata.rb

```diff
@@ -93,6 +95,10 @@ def errors_page_title
       "#{page_title}/errors"
     end
 
+    def csv_page_title
+      "#{page_title}/csv"
```

Comment:
One tiny thing I'd suggest here is making this `"#{page_title}/comparison_csv"` instead, since a bare "csv" might be confused as a reference to the CSV source for the comparison, or something...

Reply:
Oh yes, that's a good point actually. I've changed that to `/comparison_csv` in https://github.com/everypolitician/compare_with_wikidata/pull/101/commits/5dd29ea2c70e91e37914faae553f2d38083a747d.